.github/workflows/build.yml

```diff
@@ -169,6 +167,7 @@ jobs:
       cmake_preset_type: ${{needs.common_config.outputs.cmake_preset_type_resolved}}
       cibw_image_tag: ${{needs.cibw_docker_image.outputs.tag}}
       matrix: ${{needs.common_config.outputs.linux_matrix}}
+      distro: ec2
```

Comment:
Given the current Mongo fixture can't support more than 2 cores, probably don't need to run all the Python versions on EC2.

Reply:
We don't run any tests on the ec2 runners.
We only compile the wheels on ec2 runners, all tests are done on the free GH runners.

Reply:
So the `distro` is only used by part of the job, which may cause confusion.

I would either use another name for this, e.g. `compile-on-ec2`, or put a comment here.